dashboard/src/pages/databases/query.tsx

```
@@ -0,0 +1,95 @@
+          <p>{userCodeError}</p>
+        </div>
+      ) : (
+        <QueryDynamicTable mapFn={editorCodeFnString} name={name} />
+      )}
+    </div>
+  );
+}
+
+function QueryDynamicTable({ mapFn, name }: { mapFn: string; name: string }) {
+  const { useLiveQuery } = useFireproof(name);
+  const allDocs = useLiveQuery(eval(`(${mapFn})`));
```

Biome (1.9.4)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)